ports/mimxrt/boards/TEENSY41/mpconfigboard.mk

```diff
@@ -3,5 +3,8 @@ MCU_VARIANT = MIMXRT1062DVJ6A
 
 MICROPY_FLOAT_IMPL = double
 
+SRC_C = \
```

Comment:
use `+=`

Reply:
Done, even if it did not hurt, since this file is included at the top of Makefile.